Frontend/app/problem/display/displayProblem.component.ts

```diff
@@ -95,7 +128,7 @@ export class DisplayProblemComponent implements OnInit, OnDestroy {
   showNextBadStep(){
     if (this.activeBadStepCount >= this.stepLimit) {
       this.sendFailUpdate();
-      this.toastr.error("This doesn't seem quite right", "Uh Oh");
+      this.toastr.error("There is at least one mistake here or above", "Look Carefully");
```

Comment:
"There is at least one mistake present" is more extendable, in case things were to move. Generic wording wins